modules/backend/formwidgets/DatePicker.php

```diff
@@ -89,7 +96,11 @@ public function prepareVars()
 
 
         if ($value = $this->getLoadValue()) {
-            $value = $value instanceof Carbon ? $value->toDateTimeString() : $value;
+            if ($value instanceof Carbon) {
+                $value = $value->toDateTimeString();
+            } elseif ($value == $this->default) {
```

Comment:
This doesn't make much sense because this expression contradicts the actual feature. You want to specify a default, right?

```
default: "tomorrow + 3 hours 15 minutes"
```

Then

```
if (null == "tomorrow + 3 hours 15 minutes")
```

How will the value ever be applied?